srcpkgs/python3-docker/template

```diff
@@ -1,14 +1,16 @@
 # Template file for 'python3-docker'
 pkgname=python3-docker
-version=4.4.4
+version=5.0.2
 revision=1
 wrksrc="docker-${version}"
 build_style=python3-module
+make_check_target="tests/unit" # other tests fail due to needing a running docker daemon
 hostmakedepends="python3-setuptools"
-depends="python3-requests python3-six python3-websocket-client python3-docker-pycreds"
+depends="python3-requests python3-websocket-client python3-paramiko>=2.4.2"
```

Comment:
we have had `python3-paramiko>=2.4.2` since 2018, so we can drop this version restriction

Reply:
done